filters/src/main/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLogger.java

```diff
@@ -169,8 +178,10 @@ private String logProcessInstanceRecord(final Record<?> record) {
   private String logIncidentRecord(final Record<?> record) {
     final IncidentRecordValue value = (IncidentRecordValue) record.getValue();
     final StringJoiner joiner = new StringJoiner(", ", "", "");
-    joiner.add(String.format("(Element id: %s)", value.getElementId()));
-    joiner.add(String.format("(Process id: %s)", value.getBpmnProcessId()));
+    if (record.getRecordType().equals(RecordType.EVENT)) {
```

Comment:
❌ We should also log commands and command rejections. Unless we decided to only print events

Reply:
We always print every Record. For some commands however the details are empty. For example for this incident, if we receive a command no `elementId` or `bpmnProcessId` will be set.

Reply:
🔧 Aha, in that case, let's just base it on the properties themselves. If the property has a value, let's print it.

🔧 It might help to rename these methods to `logIncidentRecordValue`, to differentiate it from logging the Record, WDYT?